interfaces/portal/src/app/utils/chart.ts

```diff
@@ -1,14 +1,14 @@
 import { Chart } from 'chart.js';
 import ChartDataLabels from 'chartjs-plugin-datalabels';
 
-import { getTailwindConfig } from '~/utils/tailwind';
+import tailwindConfig from '~/../../tailwind.config';
 
-const tailwindConfig = getTailwindConfig();
 const colors = tailwindConfig.theme.colors;
 
 export const registerChartDefaults = () => {
-  Chart.defaults.font.family =
-    tailwindConfig.theme.fontFamily.display.join(', ');
+  const styles = getComputedStyle(document.documentElement);
```

Comment:
:/ This looks scary/gross/weird/evil/cunning... <scratch out whatever you feel like when "it works">.

Reply:
Yeah this is the recommended approach from the new docs https://tailwindcss.com/docs/theme#referencing-in-javascript